BurnOutSharp/ProtectionType/MediaCloQ.cs

```diff
@@ -23,7 +60,8 @@ public string CheckFilePath(string path)
         {
             var matchers = new List<PathMatchSet>
             {
-                new PathMatchSet(new PathMatch("sunncomm.ico", useEndsWith: true), "MediaCloQ"),
```

Comment:
Is the `ico` not present? Or is it somehow over-matching? If not, please keep it around with a note instead.

Reply:
The file is just a SunnComm icon, but will gladly re-add as a comment.